tutor/src/components/homework-questions.js

```diff
@@ -186,9 +188,11 @@ const ReviewExerciseCard = observer(({
     questionType = 'teacher-preview',
     styleVariant = 'points',
 }) => {
+    const isDropped = !!info.droppedQuestion;
     return (
         <QuestionPreview className="openstax-exercise-preview">
-            <QuestionHeader variant={styleVariant} className="question-header">
+            <QuestionHeader variant={styleVariant} isDropped={isDropped} className="question-header">
+                {isDropped && <DroppedQuestionIndicator model={info.droppedQuestion} size={1.6} />}
```

Comment:
DroppedQuestionIndicator will only render if `info.drop_method` is set, you shouldn't need the conditional `isDropped`

Reply:
I only need it here because it could be undefined from trying to `.find` it, but I do like that approach, maybe later I can change it to use an empty object.